### PR TITLE
Fix bug with collapsed form values

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

This bug exists in collapsed form items. When navigating to a page with filled form items in a collapse, and away, the redux was getting overriden.

This solves that issue

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2348** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2348" target="_blank">Graphite</a> 
    * **PR #2349** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2349" target="_blank">Graphite</a> 
      * **PR #2350** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2350" target="_blank">Graphite</a> 
        * **PR #2351** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2351" target="_blank">Graphite</a> 
          * **PR #2368** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2368" target="_blank">Graphite</a> 
            * **PR #2369** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2369" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2369?utm_source=stack-comment).

File: src/components/Create/components/pages/NftRewards/hooks/NftRewardsForm.ts [hunk 1]
```diff
@@ -38,11 +38,22 @@ export const useNftRewardsForm = () => {
...
 
-    return { rewards, collectionName, collectionSymbol, collectionDescription }
+    return {
+      rewards,
+      collectionName,
+      collectionSymbol,
+      collectionDescription,
+      postPayMessage: postPayModal?.content,
+      postPayButtonText: postPayModal?.ctaText,
+      postPayButtonLink: withoutHttp(postPayModal?.ctaLink),
```
Comment (tomquirk) [inline src/components/Create/components/pages/NftRewards/hooks/NftRewardsForm.ts:48]: **suggestion** can't we just make any link always include the https prefix? Confusing to have both options

**Review by tomquirk — APPROVED** [review]

File: src/components/Create/components/pages/NftRewards/hooks/NftRewardsForm.ts (hunk 1 above)
Comment (wraeth-eth) [reply src/components/Create/components/pages/NftRewards/hooks/NftRewardsForm.ts:48]: wdym?

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2369?utm_source=stack-comment).


Job Reference: tq0DBErRQ758wBw1xKNP

**Pull request merged**